instance/models/mixins/openedx_storage.py

```diff
@@ -39,6 +41,7 @@ class Meta:
     s3_access_key = models.CharField(max_length=50, blank=True)
     s3_secret_access_key = models.CharField(max_length=50, blank=True)
     s3_bucket_name = models.CharField(max_length=50, blank=True)
+    s3_region = models.CharField(max_length=50, blank=True, default=default_setting('AWS_S3_DEFAULT_REGION'))
```

Comment:
@lgp171188 It might make sense to move these fields to `S3BucketInstanceMixin`? I'm not sure why they ended up in this class in the first place; it seems like `S3BucketInstanceMixin` would be a better fit for them.

Reply:
@itsjeyd, `S3BucketInstanceMixin` was the first place I checked for these fields and since I didn't find them there, I searched further and found them in this class. I will move these fields to the mixin and test if everything works okay.